modules/ui/fields/input.js

```diff
@@ -404,16 +407,20 @@ export function uiFieldText(field, context) {
         }
     }
 
-
     function change(onInput) {
         return function() {
             var t = {};
             var val = utilGetSetValue(input);
-            if (!onInput) val = context.cleanTagValue(val);
 
-            // don't override multiple values with blank string
+            if (field.type === 'tel') {
+                val = val.replace(/^tel:\/\//, ''); // Remove 'tel://' prefix if present
+                val = val.replace(/^tel:/, ''); // Remove 'tel:' prefix if present
+            }
+    
+            if (!onInput) val = context.cleanTagValue(val);
+ 
             if (!val && getVals(_tags).size > 1) return;
-
+    
```

Comment:
We should probably perform the `tel:`-stripping after cleaning the tag value. `cleanTagValue()` removes stuff like leading and trailing whitespace, something that can easily occur if someone is copy-pasting from another website.